cpp/daal/include/services/daal_defines.h

```diff
@@ -30,7 +30,9 @@
 
 #if !(defined(_MSC_VER) || defined(__INTEL_COMPILER))
     #define __int32 int
-    #define __int64 long long int
+    #if !defined(__int64)
+        #define __int64 long long int
+    #endif
```

Comment:
Why don't we handle this here for GNU 10, but redefine in makefile? Having one place where these types are defined is much better.

Reply:
I don't quite understand what you mean.
Is it better for all compilers to define __int64 in the makefile?